docs/source/usage/cmake_example.rst

```diff
@@ -0,0 +1,25 @@
+.. highlight:: bash
+
+CMake Example
+=============
+
+You can integrate Alpaka in you project via ``find_package()`` in your ``CMakeLists.txt``. This requires, that you :doc:`install </install/instructions>` Alpaka. If you do not install Alpaka in a default path such as ``/use/local/`` you have to set the ``CMake`` argument ``-Dalpaka_ROOT=/path/to/alpaka/install``.
```

Comment:
Typo: `in you` should be `into your`.

Reply:
Sorry, i did not write the change clearly. `in your CMakeLists.txt` was fine and did not need a change. What needed a change was `You can integrate Alpaka in you project ` to be corrected as   `You can integrate Alpaka into your project` (so both a typo and a wrong preposition).

Reply:
Now it should be fixed.